feat: add `blas/base/ndarray/cgemv` (#13149)
stdlib-js/stdlib: lib/node_modules/@stdlib/blas/base/ndarray/cgemv/docs/repl.txt

{{alias}}( arrays )

>     Performs the matrix-vector operation `y = alpha*A*x + beta*y`, where

Reviewer: This equation will need be updated to actually reflect the operation. See https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/blas/base/cgemv.

>     `alpha` and `beta` are scalars, `x` and `y` are ndarrays, and `A` is

Reviewer: ```suggestion
    `alpha` and `beta` are scalars, `x` and `y` are one-dimensional ndarrays, and `A` is
```

Reviewer: You need to be more explicit here.

    an `M` by `N` matrix.

    Parameters
    ----------
    arrays: ArrayLikeObject<ndarray>
        Array-like object containing the following ndarrays:

>         - a two-dimensional input ndarray.

Reviewer: ```suggestion
        - a two-dimensional input ndarray corresponding to `A`.
```

        - first one-dimensional input ndarray.
        - second one-dimensional input/output ndarray.
        - first zero-dimensional ndarray containing a scalar constant.

>         - second zero-dimensional ndarray containing a scalar constant.

Reviewer: ```suggestion
        - a one-dimensional input ndarray corresponding to `x`.
        - a one-dimensional input/output ndarray corresponding to `y`.
        - a zero-dimensional ndarray containing a scalar constant corresponding to `alpha`.
        - a zero-dimensional ndarray containing a scalar constant corresponding to `beta`.
```

We need to be more descriptive here, as it is not clear in  your current descriptions which arrays correspond to which variables in the equation.

Reviewer: I'm guessing this comment is likely going to apply to many of your other PRs.

> 

Reviewer: Also, where is the `transpose` argument? Why is that not included here?

>         - a zero-dimensional ndarray Specifies whether `A` should be transposed, conjugate-transposed, or not transposed.

Reviewer: You need to wrap lines.

{{alias}}( arrays )
    Performs one of the matrix-vector operations `y = alpha*A*x + beta*y`,
    `y = alpha*A^T*x + beta*y`, or `y = alpha*A^H*x + beta*y`, where `alpha`
    and `beta` are scalars, `x` and `y` are one-dimensional ndarrays, and `A`
    is an `M` by `N` matrix.

    Parameters
    ----------
    arrays: ArrayLikeObject<ndarray>
        Array-like object containing the following ndarrays:

        - a two-dimensional input ndarray corresponding to `A`.
        - a one-dimensional input ndarray corresponding to `x`.
        - a one-dimensional input/output ndarray corresponding to `y`.
        - a zero-dimensional ndarray Specifies whether `A` should be transposed,

>         conjugate-transposed, or not transposed.

Reviewer: ```suggestion
        - a zero-dimensional ndarray specifying whether `A` should be
        transposed, conjugate-transposed, or not transposed.
```
